### PR TITLE
[MM-30990] Add documentation for comparison command

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR adds some documentation related to the new `comparison` command.

#### Ticket

https://mattermost.atlassian.net/browse/MM-30990

### CODE REVIEW

File: docs/comparison.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Introduction
+
+This document explains how to run an automated load-test comparison.
+
+## Prerequisites
+
+- [Terraform](https://learn.hashicorp.com/terraform/getting-started/install). The minimum supported version is 0.12.
+- AWS credentials to be used as described [here](https://www.terraform.io/docs/providers/aws/index.html#authentication)
+- A Mattermost E20 license, required to run the load-test through the [`coordinator`](coordinator.md)
```
Comment (isacikgoz) [inline docs/comparison.md:11]:
```suggestion
- AWS credentials to be used as described [here](https://www.terraform.io/docs/providers/aws/index.html#authentication).
- A valid Mattermost E20 license, required to run the load-test through the [`coordinator`](coordinator.md).
```

File: docs/comparison.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+There are a few interesting flags that the `comparison run` command supports:
+
+- `--archive`  To output all artifact (reports and graphs) into a `.zip` file.
+- `--format` Defines the format of the final output. It can be either `plain`
+    for a text based output, or `json`.
+- `--output-dir` An optional output directory where to write the output files.
+
+#### Note
+
+Depending on how it was configured, the comparison process could take hours to complete.
```
Comment (isacikgoz) [inline docs/comparison.md:76]:
```suggestion
Depending on how it was configured, the comparison process can take hours to complete.
```

File: docs/comparison_config.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+*int*
+
+The number of users to run. This is only considered if Type is "bounded"
+
+### Duration 
+
+*string*
+
+The duration of the load-test. This is only considered if Type is "bounded"
```
Comment (isacikgoz) [inline docs/comparison_config.md:69]:
```suggestion
The number of users to run. This is only considered if Type is "bounded".

### Duration 

*string*

The duration of the load-test. This is only considered if Type is "bounded".
```

**Review by isacikgoz — APPROVED** [review]
Awesome, thanks for the clear documentation @streamer45 🎉

File: docs/comparison_config.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+Possible values:
+- "mysql"
+- "postgresql"
+
+### NumUsers
+
+*int*
+
+The number of users to run. This is only considered if Type is "bounded"
```
Comment (agnivade) [inline docs/comparison_config.md:63]: Should Type be inside backticks?

File: docs/comparison_config.md (hunk 3 above)
Comment (agnivade) [inline docs/comparison_config.md:69]: Same as above.

**Review by agnivade — COMMENTED** [review]
Some minor comments.

File: docs/comparison_config.md (hunk 4 above)
Comment (streamer45) [reply docs/comparison_config.md:63]: Yes :+1:

**Review by agnivade — APPROVED** [review]

**Pull request merged**